kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/api/Configuration.java

```diff
@@ -9,9 +9,16 @@
 
 public interface Configuration {
 
-    String KUBERNETES_MASTER = "kubernetes.master";
-    String KUBERNETES_NAMESPACE = "kubernetes.namespace";
-    String KUBERNETES_DOMAIN = "kubernetes.domain";
+    //Deprecated Property Names: {
+        String KUBERNETES_MASTER = "kubernetes.master";
```

Comment:
One question why these properties are deprecated? I mean don't we need kubernetes.master in any case?